scripts/build_setup.sh

```diff
@@ -10,27 +10,32 @@ fi
 if [ "$ARCH" == "linux_i686" ]; then
   export CC="gcc -m32"
   export HOST="i686-linux-gnu"
-  export CONFIG_FLAGS="CFLAGS=-m32 CXXFLAGS=-m32 LDFLAGS=-m32"
+  export LIBHIDAPI_NAME="hidapi-libusb"
+  # export CONFIG_FLAGS="CFLAGS=-m32 CXXFLAGS=-m32 LDFLAGS=-m32"
```

Comment:
This variable prevents GCC to compile (on linux_i686). I don't know why. But doesn't seem to be needed.